lib/Controller/Library.php

```diff
@@ -589,7 +590,7 @@ function grid(Request $request, Response $response)
             $media->thumbnailUrl = '';
             $media->downloadUrl = '';
 
-            if ($media->mediaType == 'image') {
+            if ($media->mediaType == 'image' || ($media->mediaType == 'video' && file_exists($libraryLocation . $media->mediaId . '_videocover.png'))) {
```

Comment:
Can we have a new method on the media interface for this? something like `hasThumbnail()` which is default false in ModuleWidget, but the overridden in Image and now also in Video.

Would allow us to extend more easily in the future (say if someone wanted to provide a thumbnail for package html)